suite-native/device-manager/src/components/DeviceItemContent.tsx

```diff
@@ -1,82 +1,201 @@
-import React from 'react';
+import React, { ReactNode } from 'react';
```

Comment:
There is 5 components in this file. Could this be separated a little better please?